salt/orch/remove-nodes.sls

```diff
@@ -0,0 +1,46 @@
+{%- set removed_nodes = pillar.get('caasp_removed_nodes', []) -%}
+
+# Generic Updates
+update_pillar:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_pillar
+
+update_grains:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_grains
+
+update_mine:
+  salt.function:
+    - tgt: '*'
+    - name: mine.update
+    - require:
+       - salt: update_pillar
+       - salt: update_grains
+
+{%- for node_id in removed_nodes %}
+
+{{ node_id }}-stop-services:
+  salt.state:
+    - tgt: {{ node_id }}
+    - sls:
+      - container-feeder.stop
+      - kube-apiserver.stop
+      - kube-controller-manager.stop
+      - kube-scheduler.stop
+      - kube-proxy.stop
+      - kubelet.stop
+      - docker.stop
+      - etcd.stop
+
+{{ node_id }}-clean-data:
+  salt.function:
+    - tgt: {{ node_id }}
+    - name: cmd.run
+    - arg:
+      - rm /etc/pki /var/lib/docker /var/lib/kubelet /var/lib/etcd -fr
```

Comment:
I'd be more comfortable if we did a `shred` on `/etc/pki`.

Reply:
I would do these things in the appropriate files for these components. So the kubelet should have a `kubelet/removal-X` where things related to the kubelet are removed, etc. The same would apply for `certs`, `etcd`, etc...

(note that the `etcd` case is already covered in #351)

Reply:
I agree with @inercia

Reply:
Ok, I will take care about it, but as I said in comment above `shred` can operate on disk only.